SearchEngines.yml

```diff
@@ -1447,6 +1447,8 @@ Mailru:
       - windows-1251
     hiddenkeyword:
       - '/redir\?.*redir=.*/'
+      - '/.*/'
```

Comment:
Well, yes 🙂
The referrer I always get on target pages is simply "https://go.mail.ru/", although each request passes through some page which URL matches the old regex, so I left it just in case.
I guess I didn't quite understood how referrer matching in Matomo works 😅

Reply:
Sorry for the delay. If it's simply `https://go.mail.ru/`. the correct entry would be a simple `/` to match an empty url (and leave the other one untouched)

Reply:
```suggestion
      - '/redir\?.*redir=.*/'
      - '/^$/'
      - '/'
```

Reply:
Thanks! I updated the branch with suggested changes